src/components/Editor/GeneralInformation/BaseFields.vue

```diff
@@ -431,6 +431,33 @@
       </v-expand-transition>
     </v-col>
 
+    <!-- Upload logo -->
+    <v-col cols="12">
+      Logo:
+      <upload-files
```

Comment:
This field causes formValid to be false unless a logo is selected. But, not every record will have a logo, so it should also be valid if empty. 
As it stands, it's not possible to update a record's GeneralInfo without adding a logo.

Reply:
have to check it. tnx